philsloan/Club-Sandwich-Sandwich-Club
# Db setup + seed data

First pass DB setup is complete along with seed data. 

To verify:

- use mysql to run the schema.sql (creates the sandwich_db)
- run 'npm install'
- setup your .env file using the supplied .env.EXAMPLE with your username and password
- run 'npm run seed'

You can confirm the contents of the db by running the following queries
- SELECT * FROM users;
- SELECT * FROM sandwiches;
- SELECT * FROM ratings;
